content/cloud-docs/run/states.mdx

```diff
@@ -54,7 +58,7 @@ _States in this stage:_
 _Leaving this stage:_
 
 - If cost estimation succeeded or errors, the run moves to the next stage.
-- If there are no policy checks or applies, the run skips to completion (**Planned (Finished)** state).
+- If there are no policy checks or applies, the run does not continue (**Planned (Finished)** state).
```

Comment:
Hmm, this review is really turning up all kinds of underloved bits and pieces on this page 😂 "planned" and "planned and finished" are actually different states in the UI and API - "planned and finished" means that the run is over (either because it's a plan-only run, or because the apply won't run because there's no changes), while "planned" by itself means that the plan is over but there's more run to do. It looks like most instances of "planned" on this page are really referring to "planned and finished" - I'll take a pass through and update all those to be clearer.